packages/components/src/List/DisplayTable/DisplayTable.component.js

```diff
@@ -147,7 +149,14 @@ function ListHeader(props) {
 	return (
 		<tr>
 			{(isSelected && onToggleAll) && (<th />)}
-			{columns.map((column, index) => (<th key={index}>{headerContent(column, sort)}</th>))}
+			{columns.map((column, index) => (
+				<th key={index}>
+					{column.label}
+					<div aria-hidden="true" className={theme.header}>
+						{headerContent(column, sort)}
```

Comment:
Because of `aria-hidden`, if columns have `Actions`, they will be unreachable from Screen Readers :(